### DISCUSSION TITLE
https://widcard.win/posts/notes/vite-rsw

### DISCUSSION THREAD

**Discussion by giscus** [widcardw/giscus-discussion#11] [Announcements]
# https://widcard.win/posts/notes/vite-rsw

Opinionated Vite Starter Template

https://widcard.win/posts/notes/vite-rsw

**Comment by widcardw** [comment]
vite 官方文档的 [WebAssembly](https://cn.vitejs.dev/guide/features.html#webassembly) 给出了一部分样例，可以按照官方文档的方案来引入。

```ts
import init from './example.wasm?init'

init().then((instance) => {
  instance.exports.test()
})
```
